privacy/readme/CONFIGURATION.rst

```diff
@@ -0,0 +1,2 @@
+In the "Privacy", open the "Settings" menu to find and enable
```

Comment:
Put as list of actions:
```rst
#. Go to *Privacy > Settings*
#. You can enable/disable the current available features there.
```

Reply:
@Yajo please update this. there is not "settings" under "privacy"